components/List/examples/CollectionListCellExampleVanilla.m

```diff
@@ -0,0 +1,199 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MaterialIcons+ic_info.h"
+#import "MaterialInk.h"
+#import "MaterialTypography.h"
+#import "supplemental/CollectionListCellExampleVanilla.h"
+#import "supplemental/CollectionViewListCell.h"
+
+static NSString *const kReusableIdentifierItem = @"itemCellIdentifier";
+static NSString *const kExampleDetailText =
+    @"Pellentesque non quam ornare, porta urna sed, malesuada felis. Praesent at gravida felis, "
+     "non facilisis enim. Proin dapibus laoreet lorem, in viverra leo dapibus a.";
+static const CGFloat kSmallestCellHeight = 40.f;
+static const CGFloat kSmallArbitraryCellWidth = 100.f;
+
+@implementation CollectionListCellExampleVanilla {
```

Comment:
Replace "Vanilla" here and throughout with "Typical Use"

Reply:
This will make the example align more closely with our language use in other places.